test/extended/testdata/bindata.go

```diff
@@ -23785,7 +23805,8 @@ var _examplesQuickstartsRailsPostgresqlJson = []byte(`{
   },
   "message": "The following service(s) have been created in your project: ${NAME}, ${DATABASE_SERVICE_NAME}.\n\nFor more information about using this template, including OpenShift considerations, see https://github.com/openshift/rails-ex/blob/master/README.md.",
   "labels": {
-    "template": "rails-postgresql-example"
+    "template": "rails-postgresql-example",
+    "app": "rails-postgresql-example"
```

Comment:
The `app` label should be parameterized so that when templates are used in conjunction to form a single application we can make the `app` labels match.